java/arcs/core/util/CoreDispatchers.kt

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2020 Google LLC.
+ *
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ *
+ * Code distributed by Google as part of this project is also subject to an additional IP rights
+ * grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+package arcs.core.util
+
+import kotlinx.coroutines.Dispatchers
+
+/**
+ * A set of dispatchers used by Arcs for launching coroutines. These mostly mimic those
+ * in the builtin Kotlin [Dispatchers] but can be overridden via source inclusion/exclusion.
```

Comment:
How would the source inclusion/exclusion work in practice?

Reply:
Don't copybara this into G3, and instead create alternate versions in G3. Not ideal, this was mainly a suggestion for time pressure.